mcs/class/System.Data/System.Data.SqlClient/SqlDataReader.cs

```diff
@@ -1426,6 +1428,25 @@ public virtual XmlReader GetXmlReader (int i)
 			throw new NotImplementedException ();	
 		}
 
+		override public Task<T> GetFieldValueAsync<T> (int i, CancellationToken cancellationToken)
+		{
+			return base.GetFieldValueAsync<T> (i, cancellationToken);
+		}
+
+		override public Stream GetStream (int i)
+		{
+			return base.GetStream (i);
+		}
+		override public TextReader GetTextReader (int i)
+		{
+			return base.GetTextReader (i);
+		}
+
+		override public Task<bool> IsDBNullAsync (int i, CancellationToken cancellationToken)
+		{
+			return base.IsDBNullAsync (i, cancellationToken);
+		}
+
```

Comment:
yes-ish :)

They showed up in the diff because without those overrides the methods from the DbDataReader base class got used:

```diff
-public System.Threading.Tasks.Task<T> GetFieldValueAsync<T> (int i, System.Threading.CancellationToken cancellationToken)
+public System.Threading.Tasks.Task<T> GetFieldValueAsync<T> (int ordinal, System.Threading.CancellationToken cancellationToken)
-public System.IO.Stream GetStream (int i)
+public System.IO.Stream GetStream (int ordinal)
-public System.IO.TextReader GetTextReader (int i)
public System.Threading.Tasks.Task<bool> IsDBNullAsync (int ~~i~~ ordinal, System.Threading.CancellationToken cancellationToken)
```